deploy/helm/overrides.yaml

```diff
@@ -238,3 +238,9 @@ prometheus:
       - action: keep
         regex: up
         sourceLabels: [__name__]
+    # prometheus metrics
+    - url: http://fluentd:9888/prometheus.metrics
+      writeRelabelConfigs:
+      - action: keep
+        regex: 'prometheus_remote_storage_*'
```

Comment:
@abhi-sumo Could you link us the prometheus doc on this?

Reply:
synced with Abhi offline, he'll loop back with us with the docs. In the meantime he can show us the metrics in prometheus. going to merge the PR though so the metrics are available